index.html

```diff
@@ -2488,6 +2488,23 @@ <h1>DID Methods</h1>
             <a href="https://htmlpreview.github.io/?https://github.com/persistentsystems/corda-did-method/blob/master/corda_did_method.html">Corda DID method</a>
           </td>
         </tr>
+        <tr>
+          <td>
+            did:did:
```

Comment:
Have you considered did:inception as an alternative?

Reply:
I assume you mean the choice of name rather than an existing `did:inception` specification or implementation, as I am not aware of it, although perhaps it already exists - or we may already be using it without being aware of it?

`did:inception` could be good for discoverability, depending on cultural context. But `did:did` has the useful property of being able to extend an existing DID merely by prefixing "did:". With `did:inception` it would either need to have the longer prefix "did:inception:" (e.g. `did:inception:did:example:1234`), or rely on insertion rather than prefixing (e.g. `did:inception:example:1234`) - which would lose the property of the original DID being contained in the constructed DID. It seemed like a natural fit to use "did" as the method-name since DIDs begin with "did:".

We also prefer the shorter name for its memorability, and to support encoding in space constrained environments such as QR codes.

Reply:
Yes, I meant the name, but you're right, use of `did:inception` would result in early termination for nested `did:did` values.... and that would just be silly.

> support encoding in space constrained environments such as QR codes.

I assume you're using a QRCode extension to support fractal encoding of some kind to visually express the nesting advantage that `did:did` provides while keeping byte expression low... perhaps an expression such as a Sierpiński carpet? Have you considered applying the same approach to lift the stack depth limit imposed in `1.4.1.1 Denial of Service` when operating in DID-space? It would allow the expression of an effectively unbounded nesting in a `did:did` identifier by projecting DID-space into Euclidean space, with a fractal component (obviously).